release_tester/selenium_ui_test/pages/analyzers_page.py

```diff
@@ -689,7 +688,7 @@ def add_new_analyzer(self, name, ui_data_dir=None):
 
             self.tprint(f'Selecting first pipeline analyzer as Norm for {name} \n')
             if self.version_is_newer_than('3.11.99'):
-                norm = '//*[@id="chakra-modal--body-7"]/div/div[3]/div/div/div[1]/div/div/div/div[1]/div[2]'
+                norm = "(//div[contains(@class,'css-nmh171')])[3]"
```

Comment:
same as above, `css-XYZ` is not stable, can change on every build

Reply:
Taken care of.